.github/workflows/ci.yml

```diff
@@ -198,7 +198,7 @@ jobs:
             backend: tbb
             device_type: HOST
           - os: windows-2019
-            cxx_compiler: dpcpp
+            cxx_compiler: icpx
```

Comment:
Let's change it to icx or icx-cl, compiler drivers we recommend using on windows.

Reply:
Does it make sense to keep this as icpx for coverage?  We lose coverage of the workaround in the CI if we change this.

Reply:
You are right, let's leave it